libdnf/conf/config_main.cpp

```diff
@@ -371,6 +372,7 @@ ConfigMain::Impl::Impl(Config & owner) : owner(owner) {
     owner.opt_binds().add("varsdir", varsdir);
     owner.opt_binds().add("reposdir", reposdir);
     owner.opt_binds().add("debug_solver", debug_solver);
+    owner.opt_binds().add("debug_solver_dir", debug_solver_dir);
```

Comment:
I am not sure whether it is good idea to inherit it from configuration. For offline upgrade and daemon it can be confusing, because they cannot use the default. @m-blaha what do you think?

Reply:
It might make sense for such cases to have a configuration file for the daemon with some overrides. E.g. /etc/dnf/dnfdaemon.conf containing
```
[main]
debug_solver_dir = /tmp/dnfdaemon/debug_solver_data
```

Reply:
Then we have to support command configuration overrides.